src/components/Main.tsx

```diff
@@ -193,7 +203,7 @@ export function Main({
               />
             )}
           </div>
-          <div className="footer-container">
+          <div role="contentinfo" className="footer-container">
```

Comment:
Similar note [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role#:~:text=Each%20page%20should%20only%20include%20one%20contentinfo%20landmark%2C%20created%20by%20either%20using%20the%20%3Cfooter%3E%20element%20or%20by%20declaring%20role%3D%22contentinfo%22.%20contentinfo%20landmarks%20present%20in%20content%20embedded%20via%20%3Ciframe%3E%20do%20not%20count%20towards%20this%20limit.) about footers/contentinfo. Note what it says just below as well about preferring semantic HTML over roles whenever possible.

Reply:
^ alright, ignore my commebt about semantic HTML. :P I get that we can't do that.

Reply:
this article says page but i think they actually mean document... it doesnt really make sense to limit this per page if multiple documents are fine imo

Reply:
Yeah, I'd guess that if what it says about `main` applies to the `document` role, then that's true here as well. Tbh this explanation about `footer` was partly why I wasn't sure that the recommendation for `main` meant `document` role.